schemas/partners/index.ts

```diff
@@ -0,0 +1,13 @@
+import { FeatureSchemaDefinition } from "schemas/types";
+
+export const partners: FeatureSchemaDefinition = {
+    variants: {
+        group:[],
+        card:[
+            {
+                title:"[partners] Image Carousel",
```

Comment:
Cann't we use existing variants?

Reply:
Attach SS.

Reply:
New feature created:partners
![Screenshot 2023-03-13 191057](https://user-images.githubusercontent.com/101708645/224719407-57bbe33d-2980-44ee-b5b2-7ad219cca49f.png)

Reply:
Attach figma UI ss for the component.

Reply:
![Uploading Screenshot 2023-03-14 105847.png…]()

Reply:
![image](https://user-images.githubusercontent.com/101708645/224908165-3616bf72-3b7b-4e5e-99ff-edf39a7d73f2.png)